src/test/java/org/springframework/shell/SimpleShellCommandLineOptionsTest.java

```diff
@@ -0,0 +1,46 @@
+package org.springframework.shell;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+
+import org.junit.Assert;
+
+import org.junit.Test;
+import org.springframework.shell.core.JLineShellComponent;
+import org.springframework.shell.support.logging.HandlerUtils;
+
+public class SimpleShellCommandLineOptionsTest {
+
+    @Test
+    public void testPorcelain() throws IOException {
+        try {
+            String[] args = {"--porcelain"};
+            Bootstrap bootstrap = new Bootstrap(args);
+            JLineShellComponent shell = bootstrap.getJLineShellComponent();
+
+            Assert.assertTrue("isPorcelain should be true", shell.getCommandLine().isPorcelain());
```

Comment:
I'd be more interested in an end-to-end test to verify the output. is that possible?